scripts/python/gen.py

```diff
@@ -571,7 +571,8 @@ def launch(self):
             else:
                 self.config_file_path += self.args.config_file_name
 
-            if not os.path.isfile(self.config_file_path):
+            if (not self.args.osinstall and
```

Comment:
If the 'osinstall' argument isn't passed then 'self.args' won't have an 'osinstall' argument at all. To avoid an AttributeError on non-'osinstall' paths I think this conditional needs to be:
```if (not hasattr(self.args, 'osinstall') and``` ...

Reply:
Yep.  Before we commit the OSInstall items to master, I definitely want to run the normal deploy code with OSInstall merged in.